src/core/elevation/qgsprofilerenderer.h

```diff
@@ -183,6 +184,40 @@ class CORE_EXPORT QgsProfilePlotRenderer : public QObject
      */
     void render( QgsRenderContext &context, double width, double height, double distanceMin, double distanceMax, double zMin, double zMax, const QString &sourceId = QString() );
 
+    /**
+     * Returns the default line symbol to use for subsections lines.
+     *
+     * \see setSubsectionsSymbol()
+     * \since QGIS 3.44
+     */
+    static QgsLineSymbol *defaultSubSectionsSymbol() SIP_FACTORY;
```

Comment:
we're ok to **return** unique_ptrs now, sip can handle those as return values (just not as function arguments)

Reply:
That's good to know! Done.